common/djangoapps/student/tests/test_course_listing.py

```diff
@@ -87,8 +86,10 @@ def test_errored_course_regular_access(self):
         """
         Test the course list for regular staff when get_course returns an ErrorDescriptor
         """
-        course_key = SlashSeparatedCourseKey('Org1', 'Course1', 'Run1')
-        self._create_course_with_access_groups(course_key)
+        # pylint: disable=protected-access
+        mongo_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)
```

Comment:
`ErrorDescriptor`s should work with both, but it wouldn't surprise me if the test depended on mongo.